node/node.go

```diff
@@ -330,7 +326,7 @@ func (n *Node) startRPC() ([]net.Listener, error) {
 	listeners := make([]net.Listener, len(listenAddrs))
 	for i, listenAddr := range listenAddrs {
 		mux := http.NewServeMux()
-		wm := rpcserver.NewWebsocketManager(rpccore.Routes, n.evsw)
+		wm := rpcserver.NewWebsocketManager(rpccore.Routes)
```

Comment:
do you think it is not a good idea? I thought why event switch/hub should be a part of WebSocket manager (WebSocket conn)? why instead not just set event hub on RPC